src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -2,21 +2,23 @@
 :page-slug: publish-locally-developed
 :page-description: How to push local changes to the origin repository and pull them into the Koji editor, so you can publish the project.
 
-Although Koji provides a feature-rich online project editor, you might prefer to use your own suite of development tools in some cases.
-In the previous parts of this tutorial, you learned how to use Git to clone a Koji project to your development machine and how to install and launch a Node.js server to run your project locally.
+In the previous section of this tutorial, you learned how to use Git to clone a Koji project to your development machine and how to install and launch a Node.js server to run your project locally.
 If you have followed the first two parts of this tutorial, you will now have a scaffold project running on your local computer, so you can use your favorite text editor to modify it and add new features.
```

Comment:
This intro feels a bit long and redundant now. Maybe combine some of the concepts into some shorter sentences and/or leave out the restated parts. Let me know if you want to brainstorm.